pom.xml

```diff
@@ -8,6 +8,70 @@
     <artifactId>Oleg.Grabar</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <calculator-version>1.0</calculator-version>
+        <testng-version>7.4.0</testng-version>
+    </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.epam</groupId>
```

Comment:
You forget to commit .jar file :)
![image](https://user-images.githubusercontent.com/44112289/111839550-04519380-890c-11eb-9ac2-795c5ef0f557.png)

Reply:
Lol. So stupid mistake. Fixed

Reply:
Some of test cases should fail. There are some bugs in Calculator class.